docs/keyboard.rst

```diff
@@ -3,7 +3,7 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-pi-top Keyboard Input Library Reference
+pi-top Keyboard Input
```

Comment:
Not really pi-top keyboard specific. I would say this is more of a "keyboard input component", that can be used for development. Perhaps that is a way to think of grouping things?

* Using the pi-top itself (e.g. buttons, OLED, reading battery, etc.)
* Doing physical computing with pi-top [4] (PMA)
* Using peripherals (e.g. pi-topPULSE/pi-topPROTO+)
* Making use of additional helper classes (e.g. keyboard, speech?)